File: 1533.c
#include <stdio.h>

int main(void){
    int n, num, maior, maior2, indice, i;

    while(1) {
        scanf("%d", &n);
        if(n == 0)
            break;

        maior = indice = maior2 = 0;
        int v[n];
        for(i = 0; i < n; i++) {
            scanf("%d", &num);
            v[i] = num;
            if(num > maior)
                maior = num;

        }

        for(i = 0; i < n; i++) {
            if(v[i] < maior && v[i] > maior2) {
                maior2 = v[i];
                indice =  i+1;
            }
        }

        printf("%d\n", indice);
    }

    return 0;
}

File: 1533.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (true) {
            int n = sc.nextInt();
            if (n == 0)
                break;

            int maior = -1, maior2 = -1;
            int idx1 = 0, idx2 = 0;

            for (int i = 1; i <= n; i++) {
                int num = sc.nextInt();

                if (num > maior) {
                    maior2 = maior;
                    idx2 = idx1;

                    maior = num;
                    idx1 = i;
                } else if (num > maior2) {
                    maior2 = num;
                    idx2 = i;
                }
            }

            System.out.println(idx2);
        }

        sc.close();
    }
}
